docs/whatsnew/3.0.rst

```diff
@@ -235,11 +236,16 @@ For details see :ref:`chunk_reading`.
 
 .. _whatsnew-3.0-python3:
 
-Supporting only Python 3
-========================
+Python 3-only support
```

Comment:
This is still slightly confusing wording.  Maybe "Astropy now only supports Python 3"?